test/abi/Inputs/macOS/arm64/stdlib/baseline-asserts

```diff
@@ -1050,6 +1050,7 @@ _$sSP17withMemoryRebound2to8capacity_qd_0_qd__m_Siqd_0_SPyqd__GKXEtKr0_lF
 _$sSP25customPlaygroundQuickLooks01_bcD0Ovg
 _$sSP25customPlaygroundQuickLooks01_bcD0OvpMV
 _$sSP4_maxSPyxGvgZ
+_$sSP4_maxSPyxGvpZMV
```

Comment:
You should edit `abi/macOS/{arch}/stdlib.swift` instead of adding these to the baselines. Also, do we have to emit these property descriptors for every static property? Can we make these conditionally emittable?

Reply:
Thank you - I've moved them over. And yes, with this implementation, we will need to emit property descriptors for every static property.

Reply:
Similarly with the other baseline-asserts 😅

Reply:
Thank you! I've moved the rest over as well.